spooky-oysters/React-FrontEnd
# removed nested folder

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- remove the nested folder setup we had. Now our project is at the root folder. 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-removeNestedFolder
npm start
```

In Browser, navigate to:
http://bangazon.com:3000/

Steps to test:
Check that app functionality is still all operational

## Impacted Areas in Application

List general components of the application that this PR will affect:

All Files!

## Mentions @username

Tag users that need to review this code
